src/arch/riscv/kernel/vspace.c

```diff
@@ -40,6 +40,12 @@
 #include <kernel/stack.h>
 
 
+typedef enum PAGETABLE_LEVEL_E {
```

Comment:
Thanks for the information. Based on @lsf37's feedback, I don't think this `enum` is needed, since the code is already structured for the generic RISC-V page tables.